File: src/main/java/com/heybys/oddments/fooddelivery/persistence/order/OrderJpaRepository.java
package com.heybys.oddments.fooddelivery.persistence.order;

import org.springframework.data.jpa.repository.JpaRepository;

import com.heybys.oddments.fooddelivery.domain.order.Order;
import com.heybys.oddments.fooddelivery.domain.order.OrderId;

interface OrderJpaRepository extends JpaRepository<Order, OrderId> {}

File: src/main/kotlin/com/heybys/oddments/fooddelivery/persistence/order/OrderJpaRepository.kt
package com.heybys.oddments.fooddelivery.persistence.order

import com.heybys.oddments.fooddelivery.domain.order.Order
import com.heybys.oddments.fooddelivery.domain.order.OrderId
import org.springframework.data.jpa.repository.JpaRepository

internal interface OrderJpaRepository : JpaRepository<Order, OrderId>
